pkg/config/shared.go

```diff
@@ -400,6 +406,7 @@ func (c InteropClient) Fetcher(ctx context.Context, blobConf BlobConfig) (fetch.
 type ServiceBase struct {
 	Base             `name:",squash"`
 	Cluster          Cluster                `name:"cluster"`
+	Cache            Cache                  `name:"Cache"`
```

Comment:
@KrishnaIyer I overlooked this during review. The `name` should be lowercase.

Reply:
Indeed... :(